lib/chef/provider/group.rb

```diff
@@ -103,13 +103,21 @@ def compare_group
           unless members_to_be_removed.empty?
             @change_desc << "remove existing member(s): #{members_to_be_removed.join(", ")}"
           end
-        elsif new_resource.members != current_resource.members
-          @change_desc << "replace group members with new list of members"
+        elsif !group_members_match?
+          @change_desc << "replace group members with new list of members: #{new_resource.members}"
```

Comment:
do we want to join on members here: new_resource.members.join(", ")

Reply:
yeah slightly nicer output, can do.